server/docs/api/swagger.yml

```diff
@@ -172,6 +172,31 @@ paths:
           description: "Invalid input or other error"
         404:
           description: "Application not found"
+  /permits/applications/christmas-trees:
```

Comment:
Shouldn't the /christmasTreeForests tree_cost/ max number of trees be included too?